packages/api-explorer/src/components/SelectorContainer/SdkLanguageSelector.tsx

```diff
@@ -52,9 +53,25 @@ export const SdkLanguageSelector: FC = () => {
   })
 
   const handleChange = (language: string) => {
-    setSdkLanguageAction({ sdkLanguage: language })
+    navigate(location.pathname, { sdk: language.toLowerCase() })
   }
 
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search)
+    const sdkParam =
+      selectedSdkLanguage === 'All' ? null : selectedSdkLanguage.toLowerCase()
```

Comment:
NOTE: Currently working on a bug with SDK choosing 'All'. Working on ensuring the sdk parameter is removed when choosing All, and that this works when going back or forward in the browser history. Will update when fixed and added a test for choosing SDK as "All".